awx/playbooks/project_update.yml

```diff
@@ -26,7 +26,7 @@
   tasks:
 
     - name: delete project directory before update
-      command: "rm -rf {{project_path}}/*"  # volume mounted, cannot delete folder itself
+      command: "rm -rf {{project_path}}/* {{project_path}}/.* 2>&-"  # volume mounted, cannot delete folder itself
```

Comment:
You know this better than I do, so mind if I ask for an explanation of why `2>&-` is being added?

Reply:
im surpressing errors to stderr which may lead to a noisy log. I forgot to add `ignore_errors: true` to the playbook command, i will fix that with a new commit.

by example:

(without stderr redirect)

```bash
[sarsingh@fedora Documents]$ rm -Rf receptor/* receptor/.*
rm: refusing to remove '.' or '..' directory: skipping 'receptor/.'
rm: refusing to remove '.' or '..' directory: skipping 'receptor/..'
[sarsingh@fedora Documents]$ echo $?
1
[sarsingh@fedora Documents]$
```

(with stderr redirect)
```bash
sarsingh@fedora Documents]$ rm -Rf receptor/* receptor/.* 2>&-
[sarsingh@fedora Documents]$ echo $?
1
[sarsingh@fedora Documents]$
```